mkosi/manifest.py

```diff
@@ -238,6 +250,10 @@ def as_dict(self) -> dict[str, Any]:
             "config": config,
             # Describe the image content in terms of packages.
             "packages": [package.as_dict() for package in self.packages],
+            # Describe the image type (sysext, confext, image).
+            "output_format": self.context.config.output_format.value,
```

Comment:
The comment is outdated, and please put this in `config`

Reply:
Ouch : ( I am sorry, I am a bit in automatic mode.

Moved, and instead of `.value` I used `str()` as the rest of the `StrEnum`